docs/source-pytorch/fabric/guide/multi_node/cloud.rst

```diff
@@ -0,0 +1,114 @@
+:orphan:
+
+##########################
+Run in the Lightning Cloud
+##########################
+
+**Audience**: Users who don't want to waste time on cluster configuration and maintenance.
+
+
+The Lightning AI cloud is a platform where you can build, train, finetune and deploy models without worrying about infrastructure, cost management, scaling, and other technical headaches.
+In this guide, and in just 10 minutes, you will learn how to run a Fabric training script across multiple nodes in the cloud.
+
+
+----
+
+
+*************
+Initial Setup
+*************
+
+First, create a free `Lightning AI account <https://lightning.ai/>`_.
+Then, log in from the CLI:
+
+.. code-block:: bash
+
+    lightning login
+
+A page opens in your browser where you can follow the instructions to complete the setup.
+
+
+----
+
+
+***************************************
+Launch multi-node training in the cloud
+***************************************
+
+**Step 1:** Put your code inside a :class:`~lightning_app.core.work.LightningWork`:
+
+.. code-block:: python
+    :emphasize-lines: 5
+    :caption: app.py
+
+    import lightning as L
+    from lightning.app.components import LiteMultiNode
```

Comment:
Here it says still LiteMultiNode. This has to be changed separately (not this PR)